src/main/java/org/prebid/server/auction/OrtbTypesResolver.java

```diff
@@ -238,6 +245,38 @@ private JsonNode toCommaSeparatedTextNode(ObjectNode containerNode, String field
         }
     }
 
+    public void normalizeDataExtension(ObjectNode containerNode, String containerName, String nodePrefix,
+                                       List<String> warnings) {
+        final JsonNode data = containerNode.get(DATA);
+        if (data == null || data.isNull()) {
+            return;
+        }
+        final JsonNode extData = containerNode.path(EXT).path(DATA);
```

Comment:
Is it safe to get `/ext/data`? Potentially, `ext` can be null.
Not sure how `JsonNode` is processing in such cases.